public/js/reader.js

```diff
@@ -541,6 +621,50 @@ Reader.toggleArchiveOverlay = function () {
     return LRR.toggleOverlay("#archivePagesOverlay");
 };
 
+Reader.toggleFullScreen = function () {
+  // if already full screen; exit
+  // else go fullscreen
+  if (
+    document.fullscreenElement ||
```

Comment:
Can we get by just using `document.fullscreenElement` and `document.webkitFullscreenElement`?
I don't care about compatibility with older browsers (it's already broken since I use a bit of ES6 here and there), and the API seems to be supported enough without having to use vendor-specific properties (except for safari): https://caniuse.com/mdn-api_document_fullscreenelement

Reply:
some case document.fullscreenElement or webkit is need to check fullscreen
cuz some lower version chrome or firefox or edge is need that